src/main/java/uk/gov/hmcts/cmc/claimstore/controllers/IntegrationTestSupportController.java

```diff
@@ -29,6 +29,11 @@ public IntegrationTestSupportController(SupportRepository supportRepository) {
         this.supportRepository = supportRepository;
     }
 
+    @GetMapping("/trigger-server-error")
+    public void throwAnError() {
+        throw new IllegalStateException("Something really bad happened!");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AS00112)